tree/dataframe/inc/ROOT/RDF/ActionHelpers.hxx

```diff
@@ -501,7 +504,19 @@ public:
       return std::make_unique<RMergeableFill<HIST>>(*fObjects[0]);
    }
 
-   std::string GetActionName() { return "Fill"; }
+   // if the fObjects vector type is derived from TObject, return the name of the object
+   template <typename T = HIST, typename std::enable_if<std::is_base_of<TObject, T>::value>::type * = nullptr>
+   std::string GetActionName()
+   {
+      return std::string(fObjects[0]->IsA()->GetName()) + "<BR/>" + std::string(fObjects[0]->GetName());
+   }
+
+   // if fObjects is not derived from TObject, indicate it is some other object
+   template <typename T = HIST, typename std::enable_if<!std::is_base_of<TObject, T>::value>::type * = nullptr>
+   std::string GetActionName()
+   {
+      return "CustomObject";
```

Comment:
The label should mention what action is being executed, e.g.

```suggestion
      return "Fill";
```

or 

```suggestion
      return "Fill custom object";
```

Reply:
I picked the second option.